libs/utils/utils.js

```diff
@@ -1039,7 +1027,13 @@ export async function loadDeferred(area, blocks, config) {
     sampleRUM.observe(area.querySelectorAll('picture > img'));
   });
 
-  logPagePerf();
+  if (getMetadata('pageperf') === 'on') {
+    import('./logWebVitals.js')
+      .then((mod) => mod.default(getConfig().mep, {
+        delay: getMetadata('pageperf-delay') || 1000,
+        sampleRate: parseInt(getMetadata('pageperf-rate'), 10) || 50,
```

Comment:
Nit: we already have reasonable defaults in the method itself that match these ones, is there any worth in duplicating them here as well?